eacl/table.go

```diff
@@ -31,237 +74,198 @@ func (t Table) CopyTo(dst *Table) {
 		dst.cid = nil
 	}
 
-	dst.records = make([]Record, len(t.records))
-	for i := range t.records {
-		t.records[i].CopyTo(&dst.records[i])
-	}
-}
-
-// CID returns identifier of the container that should use given access control rules.
-func (t Table) CID() (cID cid.ID, isSet bool) {
-	if t.cid != nil {
-		cID = *t.cid
-		isSet = true
+	if t.records != nil {
+		dst.records = make([]Record, len(t.records))
+		for i := range t.records {
+			t.records[i].copyTo(&dst.records[i])
+		}
+	} else {
+		dst.records = nil
 	}
-
-	return
 }
 
-// SetCID sets identifier of the container that should use given access control rules.
-func (t *Table) SetCID(cid cid.ID) {
-	t.cid = &cid
+// LimitByContainer limits scope of the eACL to a given container.
+// By default, the eACL is applicable to any container.
+//
+// See also [Table.Container], [NewForContainer].
+func (t *Table) LimitByContainer(cnr cid.ID) {
+	t.cid = &cnr
 }
 
-// Version returns version of eACL format.
-func (t Table) Version() version.Version {
-	return t.version
-}
+// Container returns identifier of the NeoFS container to which the eACL scope
+// is limited. If container is not specified, second value is false meaning that
+// eACL may be applied to any container.
+//
+// See also [Table.LimitByContainer].
+func (t Table) Container() (cid.ID, bool) {
+	if t.cid == nil {
+		var zero cid.ID
+		return zero, false
+	}
 
-// SetVersion sets version of eACL format.
-func (t *Table) SetVersion(version version.Version) {
-	t.version = version
+	return *t.cid, true
```

Comment:
`t.cid` is a pointer here, we may use this fact to remove the second bool returning value. And check the result, it is nil or not. Pros and cons?